packages/serverless-contracts/src/contracts/apiGateway/features/lambdaHandler.ts

```diff
@@ -11,7 +11,7 @@ import {
 export const getApiGatewayHandler =
   <Contract extends ApiGatewayContract>(contract: Contract) =>
   (handler: SwarmionApiGatewayHandler<Contract>): ApiGatewayHandler<Contract> =>
-  async (event, context, _callback, ...additionalArgs) => {
+  async (event, context, callback, ...additionalArgs) => {
```

Comment:
As discussed together, could you add the same interface for the `EventBridgeContract`?